.github/workflows/deploy-published-releases.yaml

```diff
@@ -43,13 +43,15 @@ jobs:
         run: |-
           cp package.json LICENSE README.md build/
           cd build
+          find . -type f -path '*/*\.js.map' -exec sed -i -e "s~../src~.src~" {} +          
```

Comment:
We can use the [`sourceRoot`](https://www.typescriptlang.org/tsconfig#sourceRoot) option instead of replacing the path inside the files.

Reply:
For the plug bundle we can use [`mapRoot`](https://www.typescriptlang.org/tsconfig#mapRoot) to point to GitHub:

```
"mapRoot": "https://raw.githubusercontent.com/croct-tech/sdk-js/<tag>"
```

Reply:
> We can use the [`sourceRoot`](https://www.typescriptlang.org/tsconfig#sourceRoot) option instead of replacing the path inside the files.

The `sourceRoot` is currently treated as verbatim (absolute path), so nested files would point to invalid locations.